src/src/PluginStructs/P037_data_struct.cpp

```diff
@@ -660,7 +660,7 @@ String P037_data_struct::mapValue(String input, String attribute) {
             float inputFloat;
             float mappingFloat;

```

Comment:
Shouldn't those also be changed to double?
And thus also `string2float` changed into `validDoubleFromString`

Reply:
I'm OK with that, will fix.